### DISCUSSION TITLE
Unable to Access GitHub Education Re-Verification Application - Redirect Issue

### DISCUSSION THREAD

**Discussion by evnusi37** [community/community#134831] [GitHub Education]
### Hi everyone,

Hello GitHub Community and Education Team,

I'm experiencing a frustrating issue with the GitHub Education re-verification process, and I'm hoping to find some help or insight here.

**Issue Description:**
When attempting to re-verify my academic status for GitHub Education, I'm encountering a redirect problem that prevents me from accessing the application form.

**Detailed Steps:**
1. I received an email requesting to re-verify my academic status.
2. I click on the provided link (https://education.github.com/discount_requests/application).
3. Instead of reaching the application form, I'm redirected to the GitHub Pro payment page (https://github.com/pricing).

**Additional Information:**
- I've tried clearing browser cache/cookies, using different browsers, and accessing from various networks.
- This issue persists across all attempts.

**Questions:**
1. Is this a known issue that GitHub is currently addressing?
2. Are there any workarounds to complete the re-verification process?
3. If this is a bug, is there an estimated timeline for a fix?

**Impact:**
This issue is preventing me (and potentially other students) from maintaining access to crucial GitHub Education features needed for academic work and research.

Has anyone else encountered this problem or found a solution? Any assistance or information from the GitHub team would be greatly appreciated.

Thank you for your time and help!

![image](https://github.com/user-attachments/assets/f472a92b-0549-4d12-88ae-5a6d54a37a4c)

**Comment by Cyrillya** [comment]
Experiencing the same issue

**Comment by iamprincerajv** [comment]
The same issue with me. Can you tell me about the solution that you've got by now if any?
